src/CSharp/EasyMicroservices.AuthenticationsMicroservice.WebApi/Controllers/UserController.cs

```diff
@@ -24,13 +24,13 @@ public UsersController(IUnitOfWork unitOfWork/*, IJWTManager jwtManager*/) : bas
         }
 
         [HttpPost]
-        public async Task<MessageContract> UserHasExistsByUsername(UserHasExistsByUsernameRequestContract request)
+        public async Task<MessageContract<UserContract>> UserHasExistsByUsername(UserHasExistsByUsernameRequestContract request)
```

Comment:
It's not better to change method name too "GetUserByUserName" ?

Reply:
Sure